package.json

```diff
@@ -54,14 +52,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/MetaMask/state-log-explorer.git"
+    "url": "git+https://github.com/MetaMask/vault-decryptor.git"
```

Comment:
Eventually this should probably just be `https://github.com/MetaMask/vault-decryptor.git` for consistency with our other repos, but not a huge deal.